web/src/store/app/state.ts

```diff
@@ -0,0 +1,6 @@
+import {AppState} from "../type-defs";
```

Comment:
Pour info @blacroix  on peut faire des suggestions de patch directement dans les commentaires sur GitHub maintenant, pour les fix simples du genre :slightly_smiling_face:  Ca se fait bien le premier bouton à droite de l'onglet "Preview" quand tu commentes !

Hop :

```suggestion
import {AppState} from './type-defs';
```